environments/.env.ropsten-beta

```diff
@@ -11,3 +11,4 @@ GTAG_ID=GTM-P9S4QSF
 ZK_SPECIAL_API="ropsten-beta-api.zksync.io"
 ZK_SPECIAL_SCAN="https://ropsten-beta-scan.zksync.dev"
 ZK_NETWORK="ropsten-beta"
+APP_EXPLORER_VERSION=0.1.0
```

Comment:
I don't think we need to change this.

Reply:
removed

Reply:
I cannot find the revert of this change. Could you please double check?